src/main/java/org/skypro/exam/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package org.skypro.exam.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService{
+    Random random;
+    QuestionService questionService;
+
+    @Autowired
+    public ExaminerServiceImpl(JavaQuestionService questionService) {
```

Comment:
используем плюсы полиморфизма
```suggestion
    public ExaminerServiceImpl(QuestionService questionService) {
```